### ISSUE TITLE
Integration with preline.io

### ISSUE THREAD

**Issue by kstulgys** [TrellisCommerce/bigcommerce-cornerstone-tailwind-starter-base#10]
### Expected behaviour 
Loads preline js when setup done based on docs: https://preline.co/docs/index.html
### Actual behaviour 
Does not load js
### Steps to reproduce behaviour 
- stencil start
- npm run tailwind

**Comment by travish** [comment]
@kstulgys, just commenting to let you know I have this in my backlog.

**Comment by travish** [comment]
@kstulgys,

I'd like to take this opportunity to note that the issue reported here isn't really a [bigcommerce-tailwind-starter-base](https://github.com/TrellisCommerce/bigcommerce-tailwind-starter-base) issue, but potentially a fundamental misunderstanding of how to implement 3rd party JavaScript assets with Cornerstone.

Check out https://developer.bigcommerce.com/stencil-docs/javascript-and-event-hooks/adding-javascript

In my brief testing, I noticed there may be an important step missing from those documents. Here is what I did to get this to bundle correctly:

** All testing was done after a `git checkout -f main`. Therefore, testing was done on a full `tailwind.config.js` file.

1. `nvm use 18`
2. `npm i preline`
3. `tailwind.config.js` modifications:

```javascript
  content: [
    './templates/**/*.html',
    './node_modules/preline/dist/*.js',
  ],
```

```javascript
  plugins: [
    require('preline/plugin'),
  ],
```

4. In `assets/js/theme/global.js`, add the following at the top:

```javascript
import 'preline';
```

Note that I **did not** add the `<script>` tag as instructed by the Preline UI docs - this goes against BigCommerce best practices.

Doing the above will bundle the script into `assets/dist/theme-bundle.main.js`.
